extension/scripts/features/bank-investment/ttBankInvestment.css

```diff
@@ -0,0 +1,28 @@
+.tt-bank-investment-container > main {
+	padding: 10px;
+}
+
+.tt-bank-investment-selected-row {
```

Comment:
imo, theme-based things should define the theme (`body:not(.dark-mode)` and `body.dark-mode`)

Reply:
Also applies in quite some other spots here, won't add the comment everywhere.